### ISSUE TITLE
PyPI distgen can't locate distconf/ dir

### ISSUE THREAD

**Issue by TomasTomecek** [devexp-db/distgen#69]
I'm not sure if I'm doing something wrong. I installed distgen from pypi in my fedora:27 container, and it seems that it can't find default distro definitions:

```
[root@4b3229929025 /]# pip3 install distgen
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.
Collecting distgen
  Downloading distgen-0.20.tar.gz (57kB)
    100% |████████████████████████████████| 61kB 978kB/s
Collecting jinja2 (from distgen)
  Downloading Jinja2-2.10-py2.py3-none-any.whl (126kB)
    100% |████████████████████████████████| 133kB 1.7MB/s
Requirement already satisfied: six in /usr/lib/python3.6/site-packages (from distgen)
Collecting pyyaml (from distgen)
  Downloading PyYAML-3.12.tar.gz (253kB)
    100% |████████████████████████████████| 256kB 1.9MB/s
Collecting MarkupSafe>=0.23 (from jinja2->distgen)
  Downloading MarkupSafe-1.0.tar.gz
Installing collected packages: MarkupSafe, jinja2, pyyaml, distgen
  Running setup.py install for MarkupSafe ... done
  Running setup.py install for pyyaml ... done
  Running setup.py install for distgen ... done
Successfully installed MarkupSafe-1.0 distgen-0.20 jinja2-2.10 pyyaml-3.12

[root@4b3229929025 /]# dg --template /root/anaconda-ks.cfg
can't find configuration file 'fedora-21-x86_64.yaml'
[root@4b3229929025 /]# dg --template /root/anaconda-ks.cfg --distro fedora-27-x86_64.yaml
can't find configuration file 'fedora-27-x86_64.yaml'
[root@4b3229929025 /]# dg --template /root/anaconda-ks.cfg --distro fedora-27-x86_64
can't find configuration file 'fedora-27-x86_64'
[root@4b3229929025 /]# dg --template /root/anaconda-ks.cfg --distro fedora-26-x86_64
can't find configuration file 'fedora-26-x86_64'
[root@4b3229929025 /]# dg --template /root/anaconda-ks.cfg --distro fedora-26-x86_64.yaml
can't find configuration file 'fedora-26-x86_64.yaml'
```

They are installed though:
```
[root@4b3229929025 /]# pip3 uninstall distgen
Uninstalling distgen-0.20:
  /usr/local/bin/dg
  /usr/local/lib/python3.6/site-packages/distgen-0.20-py3.6.egg-info
  /usr/local/lib/python3.6/site-packages/distgen/__init__.py
  /usr/local/lib/python3.6/site-packages/distgen/__pycache__/__init__.cpython-36.pyc
  /usr/local/lib/python3.6/site-packages/distgen/__pycache__/commands.cpython-36.pyc
  /usr/local/lib/python3.6/site-packages/distgen/__pycache__/config.cpython-36.pyc
  /usr/local/lib/python3.6/site-packages/distgen/__pycache__/err.cpython-36.pyc
  /usr/local/lib/python3.6/site-packages/distgen/__pycache__/generator.cpython-36.pyc
  /usr/local/lib/python3.6/site-packages/distgen/__pycache__/multispec.cpython-36.pyc
  /usr/local/lib/python3.6/site-packages/distgen/__pycache__/pathmanager.cpython-36.pyc
  /usr/local/lib/python3.6/site-packages/distgen/__pycache__/project.cpython-36.pyc
  /usr/local/lib/python3.6/site-packages/distgen/__pycache__/version.cpython-36.pyc
  /usr/local/lib/python3.6/site-packages/distgen/commands.py
  /usr/local/lib/python3.6/site-packages/distgen/config.py
  /usr/local/lib/python3.6/site-packages/distgen/err.py
  /usr/local/lib/python3.6/site-packages/distgen/generator.py
  /usr/local/lib/python3.6/site-packages/distgen/multispec.py
  /usr/local/lib/python3.6/site-packages/distgen/pathmanager.py
  /usr/local/lib/python3.6/site-packages/distgen/project.py
  /usr/local/lib/python3.6/site-packages/distgen/version.py
  /usr/local/share/distgen/distconf/centos-7-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-20-i686.yaml
  /usr/local/share/distgen/distconf/fedora-20-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-21-i686.yaml
  /usr/local/share/distgen/distconf/fedora-21-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-22-i686.yaml
  /usr/local/share/distgen/distconf/fedora-22-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-23-i686.yaml
  /usr/local/share/distgen/distconf/fedora-23-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-24-i686.yaml
  /usr/local/share/distgen/distconf/fedora-24-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-25-i686.yaml
  /usr/local/share/distgen/distconf/fedora-25-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-26-i686.yaml
  /usr/local/share/distgen/distconf/fedora-26-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-27-i686.yaml
  /usr/local/share/distgen/distconf/fedora-27-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-28-i686.yaml
  /usr/local/share/distgen/distconf/fedora-28-x86_64.yaml
  /usr/local/share/distgen/distconf/fedora-rawhide-i686.yaml
  /usr/local/share/distgen/distconf/fedora-rawhide-x86_64.yaml
  /usr/local/share/distgen/distconf/lib/centos.yaml
  /usr/local/share/distgen/distconf/lib/fedora.yaml
  /usr/local/share/distgen/distconf/lib/general.yaml
  /usr/local/share/distgen/distconf/lib/rhel.yaml
  /usr/local/share/distgen/distconf/lib/rpmsystems.yaml
  /usr/local/share/distgen/distconf/rhel-7-x86_64.yaml
  /usr/local/share/distgen/distconf/rhel-7.2-x86_64.yaml
  /usr/local/share/distgen/distconf/rhel-7.3-x86_64.yaml
  /usr/local/share/distgen/templates/README
  /usr/local/share/distgen/templates/container/docker/parts.tpl
  /usr/local/share/distgen/templates/docker.tpl
  /usr/local/share/distgen/templates/general.tpl
  /usr/local/share/distgen/templates/makefile-macros.tpl
Proceed (y/n)? n
```

I continued:
```
[root@4b3229929025 /]# dg --template /root/anaconda-ks.cfg --distro /usr/local/share/distgen/distconf/fedora-27-x86_64.yaml
can't find configuration file 'fedora-27-i686.yaml'
```

This indeed seems as distgen can't locate distro defs in `/usr/local/share/distgen/distconf/`. RPM-packaged distgen has this in `/usr/share/distgen/distconf/`.

**Comment by praiskup** [comment]
This is a bit related to #66.

**Comment by bkabrda** [comment]
This was fixed by #82 and released in version 1.1.
